src/io/flutter/view/InspectorPanel.java

```diff
@@ -632,9 +636,8 @@ private void showPropertiesHelper(ArrayList<DiagnosticsNode> properties) {
         getEmptyText().setText(FlutterBundle.message("app.inspector.no_properties"));
         return;
       }
-      CompletableFuture<Void> loaded = loadPropertyMetadata(properties);
-
-      whenCompleteUiThread(loaded, (Void ignored, Throwable errorGettingInstances) -> {
+      final CompletableFuture<Void> loaded = loadPropertyMetadata(properties);
+      whenCompleteUiThread(loadPropertyMetadata(properties), (Void ignored, Throwable errorGettingInstances) -> {
```

Comment:
loadPropertymetadata is now getting called twice.
I assume that is a bug?

Reply:
Whoops.  Good catch.  Fixed.